src/modules/media/services/media.service.ts

```diff
@@ -214,6 +224,51 @@ export class MediaService {
 		return adapted;
 	}
 
+	/**
+	 * Get the object detail fields that are exposed as metadata
+	 */
+	public async findMetadataBySchemaIdentifier(
+		schemaIdentifier: string,
+		referer: string
+	): Promise<Partial<Media>> {
+		const object = await this.findBySchemaIdentifier(schemaIdentifier, referer);
```

Comment:
this will resolve the thumbnail paths
to then strip them back out before exporting it to xml

maybe we can allow refer to be string | null
and if it is null, we do not resolve thumbnail urls?

Or would that be too much of an optimization?